views/index.erb

```diff
@@ -132,6 +132,25 @@
                     </option>
                   <% end %>
                 </select>
+
+                <label class="usa-label" for="ip-auth-option">
+                  Inherited Proofing
+                </label>
+                <select name="ip_auth_option" id="ip-auth-option" class="usa-select">
+                  <% [
+                    ['', 'Disabled'],
+                    ['mocked-auth-code-for-testing', 'Enabled'],
+                    ['mocked-auth-code-for-failure', 'Simulate failure'],
+                  ].each do |value, label| %>
+                    <option value="<%= value %>"
+                            <%= 'selected="true"' if ip_auth_option == value %>
```

Comment:
It looks like the quotes are being escaped in the markup when output:

```html
<option value="mocked-auth-code-for-testing"
        selected=&quot;true&quot;
        >
  Enabled
</option>
```

It doesn't have an actual effect, since `selected` is a boolean attribute and its value doesn't matter, only its presence. We could remove the assignment value.


```suggestion
                            <%= 'selected' if ip_auth_option == value %>
```